boskos/README.md

```diff
@@ -0,0 +1,50 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
+
+## Introduction
+
+Imagine boskos is a ranch with cattle, and to perform each job in the ranch you need to hold a cattle. 
+
+The ranch is init with GCP projects, one per line, from `-config` flag
+
+Interface:
+
+* `/request?job=FOO&duration=X`: with job FOO, checkout a cattle for duration X.
```

Comment:
Unit can be anything? 1h, 30m, As long as it's a valid duration here.

Yeah my idea is to trigger janitor and clean the project. But I don't want to let the clean process block the service. Currently the checkout timeout will be same as kubekins_timeout, so assume kubetest should already handle the timeout and this is relatively safe.